xrviz/dashboard.py

```diff
@@ -140,7 +175,7 @@ def create_indexed_graph(self, *args):
 
         sel_data = sel_data.sel(**selection, drop=True)
         assign_opts = {dim: self.data[dim] for dim in sel_data.dims}
-        graph = sel_data.assign_coords(**assign_opts).hvplot.quadmesh(**graph_opts)
+        graph = sel_data.assign_coords(**assign_opts).hvplot.quadmesh(**graph_opts).opts(active_tools=['wheel_zoom', 'pan'])
```

Comment:
Box zoom

Reply:
Even if we provide both `pan` and `box_zoom` only one of these (last in the list) would be active. I have replaced `pan` with `box_zoom`.